internal/db/notification_test.go

```diff
@@ -0,0 +1,109 @@
+package db
+
+import (
+	goContext "context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/slovak-egov/einvoice/internal/entity"
+)
+
+func testInvoiceStatus(t *testing.T, id int, status string) {
```

Comment:
as we have test invoices it can be confusing
maybe name `assertInvoiceNotificationStatus` would be more clear